ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
@jzaefferer yes sorry `core.js` is indeed what i meant. In mobile core also actually includes most of the button css to avoid having most things also rely on button. However @tjvantoll and i discussed this a while back and both agree this should not be the case in UI and just things that depend on button css ( like new checkboxradio, selectmenu ) should include it.